### DISCUSSION TITLE
Clear the vertico minibuffer, so that I can paste a path in without hitting backspace a bunch of times to clear what was there.

### DISCUSSION THREAD

**Discussion by mikef656** [minad/vertico#405] [Q&A] [answered]
Would like to have a defun and binding to clear the minibuffer.  

Here is what I have tried (its simplistic, I am sure you could come up with something better)

Added this code to interactively clear the echo area (or mini-buffer not sure??) during vertico's find-file.

```
; in my .emacs just after use-package entry for vectico
    (defun my-kill-line ()
      "Remove text from the echo area by killing backwards"
      (interactive)
      (kill-line 0))
    
    (keymap-set vertico-map "C-M-g" 'my-kill-line)
```
Is there a better way to clear the minibuffer text than ```(kill-line 0)```?

**Comment by hmelman** [comment, marked as answer]
FWIW, M-backspace is backward-kill-word or C-a C-k works if there are multiple words.
